info/build.go

```diff
@@ -0,0 +1,6 @@
+package info
+
+var (
+	Version                                                                  = "Not an official release. Get the latest release from the github repo."
+	Major, Minor, Commit, BuildDate, BuiltBy, GitURL, Patch, EnterpriseToken string
+)
```

Comment:
nobody seems to be using these...

Reply:
I will remove the file and introduce it in another PR.